core/src/main/res/raw/ayp_youtube_player.html

```diff
@@ -162,5 +162,13 @@
             player.setPlaybackRate(playbackRate);
         }
 
+        function enterFullscreen() {
+            document.getElementById("youTubePlayerDOM").contentWindow.document.getElementById("movie_player").requestFullscreen();
```

Comment:
We can't use private elements ids like "movie_player" they might change at any time. Unless it's documented in the docs?

I will see if there is a way around this. Let me know if you have ideas.

Reply:
I have reverted to `toggleFullscreen` as it is the best option.